src/rust/base/src/screen.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Shift Cryptosecurity AG
```

Comment:
`rust/base/src/screen.rs`

I want to use Rust not only for the base over time, so can we structure it so that this will be easier?

`rust/src/screen.rs`
`rust/src/apps/bitboxbbase/config.rs`

etc.

Reply:
ack

Reply:
thanks for moving the files, but I think it makes more sense to structure it like I proposed, e.g. mirroring the existing directory structure.

- clearer where to add new Rust files
- bitboxbase can be treated as an app alongside btc/eth

Reply:
I wouldn't really say that base is an app. hopefully a lot of code can be shared between the base and the other targets in the future.

Reply:
true, but the api particular api calls of the base could be considered an app. i see this could be confusing though, was just going for consistency in packaging specific api calls in apps, but we can also not do that :+1: